packages/sdk-node/README.md

```diff
@@ -1,7 +1,256 @@
+(async () => {
+  // create a Node SDK object for API 3.1
+  const sdk = LookerNodeSDK.init31()
+  // retrieve your user account to verify correct credentials
+  const me = await sdk.ok(sdk.me(
+    "id, first_name, last_name, display_name, email, personal_space_id, home_space_id, group_ids, role_ids"))
+  console.log({me})
+  // make any other calls to the Looker SDK
+  const dashboards = await sdk.ok(
+    sdk.search_dashboards({title: 'My SDK dashboard'})
+  )
+  if (dashboards.length === 0) {
+    console.log('Dashboard not found')
+  }
+  const [ dashboard ] = dashboards
+  // do stuff with dashboard
+
+  ...
+
+  await sdk.authSession.logout()
+  if (!sdk.authSession.isAuthenticated()) {
+    console.log('Logout successful')
+  }
+})()
+```
+
+**NOTE**: By default, `LookerNodeSDK.init31()` and `LookerNodeSDK.init40()` will check for environment variables. Environment variables can be ignored by passing an empty string to the NodeSettings constructor.
+
+```typescript
+// Ignore any SDK environment variables for the node runtime
+const settings = new NodeSettingsIniFile('')
+const sdk = LookerNodeSDK.init40(settings)
+const sdk31 = LookerNodeSDK.init31(settings)
+```
+
+### Developing with multiple API versions
+
+Starting with Looker release 7.2, the experimental version of API 4.0 is available. To support iterative migration to API 4.0 from API 3.1, the single Looker SDK package now supports multiple API versions for the generated SDK classes. Both API 3.1 and API 4.0 are supported for Node and browser-based use.
+
+`LookerNodeSDK.init31()` `LookerBrowserSDK.init31()` and `Looker31SDK()` all initialize the API 3.1 implementation of the SDK.
+
+`LookerNodeSDK.init40()` `LookerBrowserSDK.init40()` and `Looker40SDK()` all initialize the API 4.1 implementation of the SDK.
+
+Code similar to the following can be used to develop with both the 3.1 and 4.0 SDKs in the same source file:
+
+```typescript
+import { Looker40SDK, Looker31SDK } from '@looker/sdk'
+import { NodeSession, NodeSettingsIniFile } from '@looker/sdk-node'
+
+const settings = new NodeSettingsIniFile()
+const session = new NodeSession(settings)
+const sdk = new Looker40SDK(session)
+const sdk31 = new Looker31SDK(session)
+
+const me40 = await sdk.ok(sdk.me())
+const me31 = await sdk.ok(sdk31.me()) // or sdk31.ok(sdk31.me())
+```
+
+## Using NodeSession for automatic authentication
+
+Almost all requests to Looker's API require an access token. Typically, this token is established when the `login` endpoint is called with correct API3 credentials for `client_id` and `client_secret`. When `login` is successful, the provided API3 credentials are used to look up the active **API user**.
+
+The `settings` provided to the `NodeSession` class include the base URL for the Looker instance, and the desired API version. When API requests are made, if the auth session is not yet established, `NodeSession` will automatically authenticate the **API User**. sed as that user.
```

Comment:
>>  **API User**. sed as that user.

sed?

Reply:
clearly a bad edit. It was "used" but that whole fragment after `sed` was supposed to be deleted.